raiden/ui/cli.py

```diff
@@ -528,7 +528,7 @@ def app(address,
 
     discovery_tx_cost = GAS_PRICE * DISCOVERY_REGISTRATION_GAS
     while True:
-        balance = blockchain_service.client.balance(address)
+        balance = blockchain_service.client.balance(address_decoder(address_hex))
```

Comment:
That does not make any sense. [`rpc.client.balance()`](https://github.com/raiden-network/raiden/blob/0961e75cd34fbfb69b3ddb9b2d451283bd57d884/raiden/network/rpc/client.py#L545) already accepts a bin address so plain `address` should have been fine.

And what you do here is basically take the hex, and decode it to binary which is exactly the same as should have been before.

Reply:
Yes, but `address` is only updates when given as a command line parameter. When `prompt_account` is run only the `address_hex` variable is updated.